examples/chart/teleport-kube-agent/values.schema.json

```diff
@@ -9,6 +9,7 @@
         "apps",
         "awsDatabases",
         "databases",
+        "databaseResources",
```

Comment:
In `examples/chart/teleport-kube-agent/values.yaml` we say
```
At least one of 'databases', 'awsDatabases' or 'databaseResources' must be provided
when database access is enabled. See the README for more details.
```

So I was not expecting to have both `databases` and `databaseResources` here :thinking:

Reply:
Putting those values as required only ensures the values are defined, not that they're not empty lists. By default all of those values are empty lists and pass the `values.schema.json` validation, which is terribly confusing :upside_down_face:.

I don't see how someone could "unset" them but if this happens the templating will definetly break when we're doing `{{ if .Values.database }} ...`.

I'll send a fix for `appResources`

Reply:
fixed in 4fb8a13b8a4f2e46bb3ba36cf43b383ec347a28e